integreat_cms/cms/views/translations/translations_management_view.py

```diff
@@ -0,0 +1,105 @@
+import logging
+
+from django.shortcuts import redirect, render
+from django.contrib import messages
+from django.utils.decorators import method_decorator
+from django.views.generic import TemplateView
+from django.db import transaction
+from django.utils.translation import gettext as _
+
+from ...forms import TranslationsManagementForm
+
+from ...decorators import permission_required
+
+logger = logging.getLogger(__name__)
+
+
+@method_decorator(permission_required("cms.manage_translations"), name="dispatch")
+class TranslationsManagementView(TemplateView):
+    """
+    View for showing the machine translations management options
+    """
+
+    template_name = "translations/translations_management.html"
```

Comment:
```suggestion
    #: The template to render (see :class:`~django.views.generic.base.TemplateResponseMixin`)
    template_name = "translations/translations_management.html"

    #: The context dict passed to the template (see :class:`~django.views.generic.base.ContextMixin`)
    extra_context = {"current_menu_item": "translations_management"}
```
We could define the current menu item context here to avoid the duplication